packages/blocks/src/api/collections.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * WordPress dependencies
+ */
+import { select } from '@wordpress/data';
+
+/**
+ * Returns all the block collections.
+ *
+ * @return {Object[]} Block collecti.
+ */
+export function getCollections() {
+	return select( 'core/blocks' ).getCollections();
+}
```

Comment:
Is this necessary? Can't we rely on selectors instead? (Selectors are reactive and not global functions).
I know we do the same mistake for other things like "categories", "block types"... but ideally, we don't continue the trend for new APIs.

Reply:
Happy to make the change, but I'm not sure what it should be. I just copied the other (bad) examples!

Reply:
The idea is that we don't really need this function.

components can call `select( 'core/blocks' ).getCollections()` directly in `withSelect`  or `useSelect`

Reply:
Done!